.circleci/config.yml

```diff
@@ -110,3 +110,41 @@ workflows:
           filters:
             branches:
               only: /^feature\/.*/
+      - app-center/mapping:
+          name: deploy-test-app-stg-mapping
+          file: mapping/staging/mapping.txt
```

Comment:
Looks like these paths are incorrect. I think it should be `apk/staging/mapping.txt`?

The build job is persisting the artifacts to a folder named `apk`:
https://github.com/rakutentech/android-buildconfig/blob/master/circleci/android-sdk/config.yml#L81

Reply:
Oh really? But, when I am building it locally, I can see the following path:

<img width="205" alt="Screen Shot 2021-06-08 at 12 54 13" src="https://user-images.githubusercontent.com/66930373/121122906-416b5900-c85d-11eb-87e2-ded9be89ce11.png">

Reply:
@khairul-alam-licon See the above link. When you persist to the workspace on CircleCI, you provide a path which it is persisted to. In this case, it's persisted to a folder `apk`. This is why the `appcenter/publish` jobs in this file are pointing to `apk/staging/testapp-staging.apk`.

Reply:
@corycaywood if the persist the outputs in apk folder, then why there is no middle directories like `apk/apk/staging/testapp-staging.apk` or `apk/mapping/staging/mapping.txt`? just curious..

Reply:
@munir-rakuten See the documentation for persist to workspace:
https://circleci.com/docs/2.0/configuration-reference/#persisttoworkspace

It's not very clear on the explanation, but basically it takes the `root/path` that you pass and persists it to `working_directory/path`